scripts/generate-redirects-from-migrations/README.md

```diff
@@ -0,0 +1,18 @@
+# Generate Redirects Script
+
+Renku Legacy projects can be migrated into projects for the current version of Renku. There is a problem, though. The identifier for the (Legacy) GitLab project is the GitLab identifier, and this requires access to GitLab to determine the namespace/slug. Once GitLab is decommissioned, this will no longer be possible.
+
+This script address this problem and registers redirect entries for the namespace/slug of the migrated project.
+
+
+## Running
+
+In this folder, run
+
+```
+go run .
+```
+
+You will be prompted for authentication tokens for Renku and GitLab. In both cases, admin tokens are required for the script wo work correctly.
+
+For Renku there is no way to create tokens -- the only option is to get the session cookie from the browser.
```

Comment:
```suggestion
You will be prompted for authentication credentials for Renku and GitLab. In both cases, the credentials should have admin privileges.

For Renku we expect value of the `_renku_session` cookie after the user logs in as an administrator in the browser.
```

Reply:
Can you please clarify what type of token is accepted for Gitlab. I suppose any kind? Or you only tested with a personal access token? If so just specify the type of token you tested with. We dont have to try to check / confirm that all token types will work.

Reply:
Good point. I only tested with a personal access token for the read_api scope, and I have updated the prompt to clarify this.

The personal access tokens are easy enough to create and manage, so I think this should be sufficient.